src/main/java/pl/javastart/task/Person.java

```diff
@@ -0,0 +1,87 @@
+package pl.javastart.task;
+
+public class Person {
+    private static final int MINIMAL_CHAR = 2;
+    private static final int INCORRECT_AGE = 1;
+
+    private String firstName;
+    private String lastName;
+    private int age;
+    private String pesel;
+
+    public Person(String firstName, String lastName, int age, String pesel) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.age = age;
+        this.pesel = pesel;
+        checkNullFirstName();
+        checkNullLastName();
+        checkNumberOfChar(firstName);
+        checkNumberOfChar(lastName);
+        checkAge(age);
```

Comment:
Najpierw sprawdzamy warunki wstępne, a dopiero jeżeli są poprawne, to robimy przypisanie.
W setterach masz tę samą logikę, więc wykorzystaj te settery w tym miescu do ustawiania pól